mysite/settings.py

```diff
@@ -23,6 +24,8 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = config('SECRET_KEY', default='secret-key')
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = '258892222487-e3lapeoqq0tem2k5oi21asccebpgv4cv.apps.googleusercontent.com' 
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = 'QjQaLqQFiP_Hmgfv3lPCr9sJ'
```

Comment:
please try to add it in .env and use config, we should not push the secret key to our repo